src/secp256k1/src/ecmult_gen_impl.h

```diff
@@ -150,7 +150,10 @@ static void secp256k1_ecmult_gen(const secp256k1_ecmult_gen_context *ctx, secp25
         secp256k1_ge_from_storage(&add, &adds);
         secp256k1_gej_add_ge(r, r, &add);
     }
+    /* see also: https://github.com/bitcoin-core/secp256k1/pull/485 */
+#ifndef __clang_analyzer__
     bits = 0;
```

Comment:
secp256k1 is maintained via subtree

Reply:
Makes sense, however notice that the referenced PR basically says that libsecp256k1 is not yet updated either. So this is just meant to quieten the static analyzer.

Reply:
Ok, I addressed this now with a separate copy of the diff in a `patch/` subdirectory.